src/content/docs/guides/JSON/1-json_reading.mdx

```diff
@@ -3,7 +3,7 @@ title: Reading JSON Data in SplashKit
 description: After understanding the basics of JSON in SplashKit, this part of the tutorial focuses on how to read and parse JSON data. Reading JSON data is essential for game development tasks such as loading game settings, level configurations, or player data.
 category: Guides
 author: Jonathan Tynan and Yuyang Yang
-lastupdated: Sep 14 2024
+lastupdated: Mep 14 2024
```

Comment:
Thanks for making those changes! They feel like nice functional changes.

Only one minor thing that's happened perhaps accidentally during the update was the date - you've now got Mep instead of Sep :)